openlibrary/templates/lists/header.html

```diff
@@ -61,6 +61,8 @@ <h1>$:_('<a href="%(href)s">%(name)s</a> / Lists', href=doc.key, name=doc.name)<
 
 $elif type == "/type/user":
     $var title: $doc.displayname
+    $if days_since(doc.created) < 30:
+       $ putctx('robots', 'noindex')
```

Comment:
It looks like these changes are from an unrelated PR.  Doing a rebase will correct this.  Instructions can be found [here](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-updates-to-your-pull-request), if needed.

Reply:
@jamesachamp I am sorry for these commits, I will try to fix it up

Reply:
@jamesachamp Done :)